ivy/functional/frontends/torch/spectral_ops.py

```diff
@@ -13,11 +12,25 @@ def bartlett_window(
     device=None,
     requires_grad=False
 ):
-    # this implementation is based on scipy.signal.windows.bartlett
```

Comment:
Are there any failures that were originating from the `if` check in this function? If not, I think we shouldn't remove the check. Secondly, we aren't really adding docstrings for frontend functions because users can view those from the docstrings of the source framework anyway 😄